reporter/src/main/kotlin/reporters/opossum/OpossumReporter.kt

```diff
@@ -483,20 +481,6 @@ class OpossumReporter : Reporter {
                 OpossumSignal(source, comment = issue.toString(), followUp = true, excludeFromNotice = true)
             addSignal(signal, paths.map { resolvePath(it) }.toSortedSet())
         }
-
```

Comment:
Should this be marked as a breaking change in the commit message?

Reply:
I decided for not doing so, as users do not need to make any alignments. 
I considered it basically under the label "change slightly how the UI displays things", for which
I believe "breaking" is not needed. What do you think?

Reply:
I'm fine either way. I just wanted to highlight it, also towards @maxhbr.